lib/buttons-and-links/KExternalLink.vue

```diff
@@ -70,6 +72,13 @@
         type: String,
         required: false,
       },
+      /**
+       * If provided, opens link in new tab
+       */
+      openInNewTab: {
+        type: Boolean,
+        required: false,
+      },
```

Comment:
```suggestion
      openInNewTab: {
        type: Boolean,
        default: false,
      },
```

Generally in prop definitions you should provide one of either `required: true`, or provide a `default`.

Providing a `default` value ensures the prop value is well-defined when nothing is provided from outside, and makes setting `required: false` superfluous.